imap_processing/mag/l0/decom_mag.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from pathlib import Path
+
+from bitstring import ReadError
+from space_packet_parser import parser, xtcedef
+
+from imap_processing import imap_module_directory
+from imap_processing.ccsds.ccsds_data import CcsdsData
+from imap_processing.mag.l0.mag_l0_data import MagL0, Mode
+
+logging.basicConfig(level=logging.ERROR)
+logger = logging.getLogger(__name__)
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
# logger.setLevel(logging.ERROR)
```
warning level is default, so we'll get the ERRORs still and I think we want them as well? So, I'd suggest either removing the level, or setting it directly on this logger itself.